README.md

```diff
@@ -19,18 +19,32 @@ Need additionally:
 * R package: [testthat](https://cran.r-project.org/web/packages/testthat/index.html) (optional)
 * [Git](https://git-scm.com/)
 
-
 ## Install the romero.gateway R package
 
-### Windows
+### CRAN (recommended)
+
+Install the package from CRAN:
+```
+  install.packages('romero.gateway')
+```
+
+### From package file
+
+If you need to install an older version of the package, because your OMERO server has an older version.
```

Comment:
```suggestion
If you need to install an older version of the package because your OMERO server has an older version,
you can install directly from the package file.
```